src/Swarm/Doc/Pedagogy.hs

```diff
@@ -157,7 +161,10 @@ generateIntroductionsSequence =
 loadScenarioCollection :: IO ScenarioCollection
 loadScenarioCollection = simpleErrorHandle $ do
   entities <- ExceptT loadEntities
-  (_, loadedScenarios) <- liftIO $ loadScenariosWithWarnings entities
+  (failures, loadedScenarios) <- liftIO $ loadScenariosWithWarnings entities
+  when (notNull failures) $
+    liftIO $
+      hPutStrLn stderr "Loading failures: " >> mapM_ (T.putStrLn . prettyFailure) failures
```

Comment:
Should the "Loading failures:" heading and the individual error lines both be sent to `stderr`?

Reply:
Yes, well spotted. 👍 The pedagogy output was pretty long, so I was thinking about redirecting `stdout` to `/dev/null` so I could see the failures only. But by then I fixed the failures. 😄